<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShowRangeSections</class>
 <widget class="QWidget" name="ShowRangeSections">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>614</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>441</width>
     <height>521</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>341</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Диапазон секций</string>
   </property>
   <widget class="QComboBox" name="comboBoxFromSection">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxToSection">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>40</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>По</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>С</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButtonChooseRange">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>50</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Выбрать</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
